spec/unit/plugins/digital_ocean_spec.rb

```diff
@@ -160,44 +154,13 @@
 
   describe "without digital_ocean hint file" do
     before do
-      allow(File).to receive(:exist?).with(hint_path_nix).and_return(false)
-      allow(File).to receive(:exist?).with(hint_path_win).and_return(false)
-    end
-
-    describe "with the /etc/digitalocean file" do
```

Comment:
I'm not sure that these describe blocks should be removed. Is this code path tested elsewhere?